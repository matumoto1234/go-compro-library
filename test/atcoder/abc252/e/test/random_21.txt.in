59 87
23 52 1
8 9 66
17 18 32770
4 33 1
15 16 8194
21 50 1
18 19 65538
11 39 1
19 48 1
17 45 1
16 45 1
21 49 1
14 15 4098
22 50 1
27 28 33554434
29 57 1
20 49 1
3 32 1
3 4 4
12 41 1
12 40 1
25 54 1
18 46 1
2 3 3
5 33 1
5 6 10
27 56 1
20 21 262146
4 5 6
15 44 1
23 24 2097154
9 10 130
18 47 1
22 23 1048578
16 44 1
19 20 131074
6 35 1
13 42 1
8 36 1
11 40 1
25 53 1
14 42 1
10 11 258
27 55 1
29 58 1
4 32 1
28 56 1
26 55 1
20 48 1
30 59 1
16 17 16386
21 22 524290
9 38 1
9 37 1
1 30 268435458
13 14 2050
28 29 67108866
7 8 34
26 27 16777218
12 13 1026
10 38 1
6 7 18
24 53 1
7 35 1
10 39 1
15 43 1
7 36 1
22 51 1
29 30 134217730
14 43 1
24 25 4194306
8 37 1
24 52 1
17 46 1
30 58 1
3 31 1
11 12 514
6 34 1
13 41 1
28 57 1
23 51 1
19 47 1
25 26 8388610
5 34 1
1 59 1
26 54 1
2 31 1
