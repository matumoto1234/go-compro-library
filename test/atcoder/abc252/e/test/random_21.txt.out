0
58
56
54
52
50
48
46
44
42
40
38
36
34
32
30
28
26
24
22
20
18
16
14
12
10
8
6
4
2
57
55
53
51
49
47
45
43
41
39
37
35
33
31
29
27
25
23
21
19
17
15
13
11
9
7
5
3
1
